src/webgl/buffer.js

```diff
@@ -293,6 +294,11 @@ export default class Buffer extends Resource {
     return this.gl.getIndexedParameter(binding, index);
   }
 
+  // returns number of elements in the buffer
+  get elementcount() {
```

Comment:
Lets be consistent. If a method `getElementCount()`.

If we want to add getters, we should do it consistently and not just for one accessor.